run-tests-with-build-optaplanner-quickstarts-parent/optaplanner-quickstarts-project-sources-test/pom.xml

```diff
@@ -58,10 +53,7 @@
             skipped (when returns false or throws error) -->
           <selectorScript>${maven.modules.resolution.selector.script.name}</selectorScript>
           <pomIncludes>
-            <!-- includes everything, filtering is done in resolve-includes.groovy script
-            which adds invoker-run.groovy script containing 'return false' near each pom.xml
-            that should be excluded.-->
-            <pomInclude>**/pom.xml</pomInclude>
+            <include>${invoker.pom.include}</include>
```

Comment:
selectorScript above is not needed anymore (when the resolve-includes was removed).